test/test_builtin_tuner/run.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
```

Comment:
I'm not sure shall we add license here....@quanlu zhang

Reply:
yes, could add license here